contracts/SportMarkets/Parlay/ParlayVerifier.sol

```diff
@@ -12,24 +12,29 @@ import "../../interfaces/IStakingThales.sol";
 import "../../interfaces/IReferrals.sol";
 import "../../interfaces/ICurveSUSD.sol";
 import "../../interfaces/ITherundownConsumer.sol";
+import "../../interfaces/IParlayPolicy.sol";
+
+// import "hardhat/console.sol";
 
 contract ParlayVerifier {
     uint private constant ONE = 1e18;
+    uint private constant ONE_PERCENT = 1e16;
```

Comment:
as I said for the future all our contracts have this const variable so i think it need some kind of Utils (same as example TheRundownConsumer address so we set it in 5 places)